broker/src/main/java/org/apache/rocketmq/broker/processor/PopReviveService.java

```diff
@@ -198,12 +198,11 @@ public PullResult getMessage(String group, String topic, int queueId, long offse
                     break;
                 case NO_MATCHED_MESSAGE:
                     pullStatus = PullStatus.NO_MATCHED_MSG;
-                    POP_LOGGER.warn("no matched message. GetMessageStatus={}, topic={}, groupId={}, requestOffset={}",
+                    POP_LOGGER.info("no matched message. GetMessageStatus={}, topic={}, groupId={}, requestOffset={}",
```

Comment:
`NO_MATCHED_MESSAGE` means the pulled messages are filtered, so I think `info` is enough, maybe debug is a better level.

Reply:
OK, let's downgrade it to debug.